src/include/migraphx/serialize.hpp

```diff
@@ -171,6 +171,9 @@ void from_value(const value& v, T& x)
     detail::from_value_impl(rank<7>{}, v, x);
 }
 
+void value_to_json_string(const value& val, std::string& str);
+void value_from_json_string(const std::string& str, value& val);
```

Comment:
This should be in a different header. And the function signatures should be:

```cpp
std::string to_json(const value& v);
value from_json(const std::string& str);
```

Reply:
fixed